episodes/severity-static.Rmd

```diff
@@ -0,0 +1,640 @@
+
+:::::::::::::::::::::::::::::::::::::::: challenge
+
+Load the `ebola_kikwit_1995` from `{outbreaks}` and estimate the delay-adjusted CFR using the appropriate distribution delay. Then:
+
+- Compare the naive and the delay-adjusted CFR solutions!
+
+:::::::::::::::::::: hint
+
+- Find the appropriate `<epidist>` object!
+
+::::::::::::::::::::
+
+:::::::::::::::::::: solution
+
+```{r}
+library(outbreaks)
+
+outbreaks::ebola_kikwit_1995 %>% glimpse()
+
+outbreaks::ebola_kikwit_1995 %>%
+  rename(
+    cases = onset,
+    deaths = death
+  ) %>%
+  cfr::cfr_static(
+    delay_density = function(x) density(onset_to_death_ebola, x)
+  )
+```
+
+::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::
+
+
+## An early-stage CFR estimate
+
+We just found that the difference between the _naive_ and _delay-adjusted_ CFR estimates using `cfr_static()` are _just slightly_ different. So, what is the main difference?!
+
+The main benefit of using `cfr_static()` is to reduce the time to get an _delay-adjusted CFR_ estimate closer to the overall CFR of the epidemic. 
+
+`cfr_static()` can assess the virulence of an emerging infectious disease at an epidemic's **early stage**. This is particularly useful when an _epidemic curve of confirmed cases_ is the only data available (i.e. when individual data from onset to death are unavailable, especially during the early stage of the epidemic). Nishiura et al., 2009 depict this in the figures below.
+
+Figures A and B show the cumulative numbers of cases and deaths of SARS, and Figure C shows the observed (biased) CFR estimates as a function of time, i.e. the ratio of the cumulative number of cases to deaths at time $t$. Due to the delay from the onset of symptoms to death, the biased estimate of CFR at time $t$ underestimates the realised CFR at the end of an outbreak (i.e. 302/1755 = 17.2 %). 
+
+![Observed (biased) confirmed case fatality ratio of severe acute respiratory syndrome (SARS) in Hong Kong, 2003.](fig/cfr-pone.0006852.g003-fig_abc.png)
+
+Nevertheless, even by only using the observed data for the period March 19 to April 2, `cfr_static()` can yield an appropriate prediction (Figure D), e.g. the delay-adjusted CFR at March 27 is 18.1 % (95% CI: 10.5, 28.1). An overestimation is seen in the very early stages of the epidemic, but the 95% confidence limits in the later stages include the realised CFR (i.e. 17.2 %).
+
+![Early determination of the delay-adjusted confirmed case fatality ratio of severe acute respiratory syndrome (SARS) in Hong Kong, 2003.](fig/cfr-pone.0006852.g003-fig_d.png)
+
+We can explore this behaviour using the `cfr_rolling()` function. With `tail()`, we show that the latest CFR estimates are equal to the `cfr_static()` outputs:  
+
+```{r}
+# Calculate the rolling daily naive CFR
+rolling_cfr_naive <- cfr::cfr_rolling(data = ebola1976)
+
+tail(rolling_cfr_naive)
+```
```

Comment:
I would explain this as a utility function that automatically calculates CFR for each day of the outbreak with the data available up to that day, saving the user time.

Reply:
Added with emphasis as callout in https://github.com/epiverse-trace/tutorials-middle/pull/1/commits/fab45c199a87b1e164cdf0d1d69d63e26963b3a7

Reply:
Okay, thanks!